src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,19 @@
 package mate.academy
 
+import mate.academy.model.Figure
+import mate.academy.service.FigureSupplier
+
 fun main() {
-    // draw all figures here
+    val figureSupplier = FigureSupplier()
+    val figures = mutableListOf<Figure>()
+
+    for (i in 1..3) {
```

Comment:
Consider refactoring to find more flexible solution. F.e. in case if you need get list for 100 items or 1000, every time range in the loop should be changed, which is not optimal solution